scripts/findAndMatch.js

```diff
@@ -0,0 +1,319 @@
+  artists.style.cssText = "visibility: visible";
+  instruments.style.cssText = "visibility: visible";
+  cheatCounter();
+}
+
+//Event listeners and draggable assignments
+for (var i = 1; i < artists.children.length; i++) {
+  artists.children[i].draggable = true;
+}
+
+for (var i = 1; i < artists.children.length; i++) {
+  artists.children[i].addEventListener("dragstart", startDrag);
+}
+
+for (var i = 1; i < instruments.children.length; i++) {
+  instruments.children[i].addEventListener("dragover", dragOver);
+}
+
+for (var i = 1; i < instruments.children.length; i++) {
+  instruments.children[i].addEventListener("drop", dropDrag);
+}
+
+//Drag and drop functionality
+var holderItem;
+
+function startDrag() {
+  holderItem = event.target;
+  console.log("Drag started");
+}
+
+function dragOver() {
+  event.preventDefault();
+  console.log("Drag allowed");
+}
+
+var artistsArr = Array.from(artists.children);
+
+function dropDrag() {
+  event.preventDefault();
+  if (
+    (holderItem.className === "guitarist" && event.target.id === "guitar") ||
+    (holderItem.className === "vocalist" && event.target.id === "mic") ||
+    (holderItem.className === "bassist" && event.target.id === "bass") ||
+    (holderItem.className === "drummer" && event.target.id === "drums")
+  ) {
+    artistsArr.splice(artistsArr.indexOf(holderItem.className), 1);
+    event.target.appendChild(holderItem);
+    instruments.appendChild(p).innerHTML = "Correct";
+    p.style.cssText = "color: darkseagreen;";
+    if (artists.children.length <= 1) {
+      instruments.appendChild(p).innerHTML =
+        "CONGRATULATIONS!" +
+        "<br>" +
+        "Please select your prize from the instruments above";
+      clearInterval(timer);
+      selectPrize();
+      p.style.cssText = "color: darkseagreen; font-size: 26px;";
+      cheat.style.cssText = "visiblity: hidden";
+    }
+  } else {
+    instruments.appendChild(p).innerHTML = "Incorrect";
+    p.style.cssText = "color: indianred;";
+  }
+  console.log("Drag dropped");
+}
+
+//Cheat / hover hint
+var cheat = document.getElementById("cheat");
+var titleValues = [];
+
+for (var i = 1; i < artistsArr.length; i++) {
+  titleValues.push(artistsArr[i].title);
+}
+
+for (var i = 1; i < artistsArr.length; i++) {
+  artistsArr[i].title = "";
+}
+
+function whatInstr() {
+  for (var i = 1; i < artistsArr.length; i++) {
+    artistsArr[i].title = titleValues[i - 1];
+  }
+}
+
+function cheatCounter() {
+  timer = setInterval(cheater, 20000);
+}
+
+function cheater(a) {
+  a = confirm(
+    "Feeling unconfident? Well, either read more about these artists or go the easy way and cheat?" +
+      "\n \n" +
+      "OK = Yes, I wanna cheat >: )" +
+      "\n" +
+      "Cancel = Imma do this on my own!"
+  );
+  if (a == true) {
+    clearInterval(timer);
+    alert("Let the IDDQD begin");
+    msg3.innerHTML +=
+      '<button id = "hintBtn" class = "btn">HOVER<br><span><img src = "img/devil.svg"></span><br>HINTS</button>';

```

Comment:
Instead of writing these long string inside the code I would try to extract them in variables so for example: 
```
var hintButton = '<button id = "hintBtn" class = "btn">HOVER<br><span><img src = "img/devil.svg"></span><br>HINTS</button>';
msg3.innerHTML += hintButton
```

Also if you need to have dynamic stuff in your components (buttons, paragraphs etc.) you can declare them as functions:
```
function createButton(buttonText) {
  return '<button id = "hintBtn" class = "btn">' + buttonText + '<br><span><img src = "img/devil.svg"></span><br>HINTS</button>'
}
//usage msg3.innerHTML = createButton("Your custom text here")
//would render <button id = "hintBtn" class = "btn">Your custom text here"<br><span><img src = "img/devil.svg"></span><br>HINTS</button>

```